TestConcurrentHubClone failure

https://travis-ci.com/getsentry/sentry-go/jobs/270853268

```
$ go test -race
--- FAIL: TestConcurrentHubClone (0.00s)
    hub_test.go:404: 
        got: []sentry.TestEvent{sentry.TestEvent{Message:"Hello from goroutine! #2", Tags:map[string]string{"secretTag":"go#2"}}, sentry.TestEvent{Message:"Hello from goroutine! #3", Tags:map[string]string{"secretTag":"go#3"}}}
        want: []sentry.TestEvent{sentry.TestEvent{Message:"Hello from goroutine! #1", Tags:map[string]string{"secretTag":"go#1"}}, sentry.TestEvent{Message:"Hello from goroutine! #2", Tags:map[string]string{"secretTag":"go#2"}}, sentry.TestEvent{Message:"Hello from goroutine! #3", Tags:map[string]string{"secretTag":"go#3"}}}
FAIL
exit status 1
FAIL	github.com/getsentry/sentry-go	0.063s
```

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -12,3 +12,7 @@ issues:
   exclude:
     - "not declared by package utf8"
     - "unicode/utf8/utf8.go"
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - prealloc

--- a/hub_test.go
+++ b/hub_test.go
@@ -3,9 +3,11 @@ package sentry
 import (
 	"context"
 	"fmt"
-	"sort"
 	"sync"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 )
 
 func setupHubTest() (*Hub, *Client, *Scope) {
@@ -389,16 +391,17 @@ func TestConcurrentHubClone(t *testing.T) {
 		},
 	}
 
-	got := make([]TestEvent, len(transport.Events()))
-	for i, event := range transport.Events() {
-		got[i] = TestEvent{
+	var got []TestEvent
+	for _, event := range transport.Events() {
+		got = append(got, TestEvent{
 			Message: event.Message,
 			Tags:    event.Tags,
-		}
+		})
 	}
-	sort.Slice(got, func(i int, j int) bool {
-		return got[i].Message < got[j].Message
-	})
 
-	assertEqual(t, got, want)
+	if diff := cmp.Diff(want, got, cmpopts.SortSlices(func(x, y TestEvent) bool {
+		return x.Message < y.Message
+	})); diff != "" {
+		t.Errorf("Events mismatch (-want +got):\n%s", diff)
+	}
 }

--- a/hub_test.go
+++ b/hub_test.go
@@ -355,7 +355,6 @@ func TestConcurrentHubClone(t *testing.T) {
 		// Mutate hub in the main goroutine.
 		hub.PushScope()
 		hub.PopScope()
-		hub.BindClient(nil)
 		hub.BindClient(client)
 		// Clone scope in a new Goroutine as documented in
 		// https://docs.sentry.io/platforms/go/goroutines/.

